Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -157,6 +169,13 @@ - (instancetype)initWithApp:(FIRApp *)app
       [sessions registerWithSubscriber:self];
     }
 
+    if (remoteConfig) {
+      FIRCLSDebugLog(@"Registering RemoteConfig SDK subscription for rollouts data");
+
+      _remoteConfigManager = [[FIRCLSRemoteConfigManager alloc] initWithRemoteConfig:remoteConfig];
+      [remoteConfig registerRolloutsStateSubscriber:self for:@"firebase"];
```

Comment:
Just leaving a note for us to import `"firebase"` from the interop in the future